src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -68,6 +68,10 @@ internal sealed partial class TexlBinding
         // Maps Ids to Info, where the Id is an index in the array.
         private readonly object[] _infoMap;
 
+        // Call nodes which do not appear in the source code
+        // For example, $"" (interpolated string) maps to a call to Concatenate
+        private readonly CallNode[] _compilerGeneratedCallNodes;
```

Comment:
>_compilerGeneratedCallNodes [](http://example.com/codeflow?start=36&length=27)

How will we use this?

Reply:
JsTranslator often queries the call info.

Reply:
This is effectively additional information similar to _infoMap, but infoMap's length is fixed + locked after the parser. whereas these new nodes are created in the binder after the parse, so we need a separate place to store them.

StrInterpolation needs both a CallInfo and CallNode.